src/core/Akka.NodeTestRunner/Akka.NodeTestRunner.csproj

```diff
@@ -75,10 +75,6 @@
       <Project>{A8AA2D7E-3D35-44DF-AF92-80A2C39C1F4D}</Project>
       <Name>Akka.Logger.NLog</Name>
     </ProjectReference>
-    <ProjectReference Include="..\Akka.MultiNodeTests\Akka.MultiNodeTests.csproj">
-      <Project>{f0781bea-5ba0-4af0-bb15-e3f209b681f5}</Project>
-      <Name>Akka.MultiNodeTests</Name>
-    </ProjectReference>
```

Comment:
No longer need to reference Akka.MultiNodeTests (which was a hack we used for putting the test assemblies into the same folder as the test runner) since they can be run from anywhere now